tests/factories/github/__init__.py

```diff
@@ -0,0 +1,81 @@
+import datetime
+import json
+from pathlib import Path
+
+from openverse_catalog.dags.maintenance.pr_review_reminders.pr_review_reminders import (
+    COMMENT_MARKER,
+    Urgency,
+)
+
+
+def _read_fixture(fixture: str) -> dict:
+    with open(Path(__file__).parent / f"{fixture}.json") as file:
+        return json.loads(file.read())
+
+
+def _make_label(priority: Urgency) -> dict:
+    return {"name": f"priority: {priority.label}"}
+
+
+def walk_backwards_in_time_until_weekday_count(today: datetime.datetime, count: int):
+    test_date = today
+    weekday_count = 0
+    while weekday_count < count:
+        test_date = test_date - datetime.timedelta(days=1)
+        if test_date.weekday() < 5:
+            weekday_count += 1
+
+    return test_date
+
+
+_pr_count = 1
+
+
+def make_pull(priority: Urgency, past_due: bool) -> dict:
```

Comment:
Do you mind adding a docstring here describing what this function does? I'm having a hard time following everything it's referencing.